### PR TITLE
feature/add missing specs + rubocop fixes

### PR DESCRIPTION
**Pull request by atytko**
- Added some missing specs
- Added rubocop fixes and adjusted configs
- Adjusted routes

### CODE REVIEW

File: app/policies/cinema_hall_policy.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class CinemaHallPolicy < ApplicationPolicy
   attr_reader :user, :cinema_hall
 
   def initialize(user, cinema_hall)
+    super
```
Comment (atytko) [inline app/policies/cinema_hall_policy.rb:7]: Rubocop suggested that change throwing: _Lint/MissingSuper: Call super to initialize state of the parent class._ I am not sure if it is really needed in this case. 🤔

File: .rubocop.yml [hunk 2]
```diff
@@ -15,3 +15,17 @@ AllCops:
     - 'config/**/*'
     - 'spec/rails_helper.rb'
 
+Metrics/MethodLength:
+  Max: 15
+
+Metrics/BlockLength:
+  ExcludedMethods: ['describe', 'context']
```
Comment (wmaciejak) [inline .rubocop.yml:22]: Nice, I didn't knew that there is option for that 👏

File: app/policies/cinema_hall_policy.rb (hunk 1 above)
Comment (atytko) [reply app/policies/cinema_hall_policy.rb:7]: Corrected, thanks :) I must have misunderstood the following example in the documentation:

_class PostPolicy
  attr_reader :user, :post

  def initialize(user, post)
    @user = user
    @post = post
  end

  def update?
    user.admin? || !post.published?
  end
end_
[](https://github.com/varvet/pundit)

Comment (jandudulski) [reply app/policies/cinema_hall_policy.rb:7]: In the mentioned example they don't use base class (like `ApplicationPolicy` in your example) so they have to define `initialize` method.

Comment (wmaciejak) [reply app/policies/cinema_hall_policy.rb:7]: @atytko consider if you really need `initialize` method if everything is already defined in [ApplicationPolicy](https://github.com/atytko/monte-cinema/blob/main/app/policies/application_policy.rb#L6)

This comment is applicable to all policies.

Comment (atytko) [reply app/policies/cinema_hall_policy.rb]: You are right, I don't need it at all. :) I have removed the `initialize` methods inside my policies.

**Review by wmaciejak — APPROVED** [review]

**Pull request merged**